filter_plugins/oo_filters.py

```diff
@@ -375,6 +375,13 @@ def oo_split(string, separator=','):
     return string.split(separator)
 
 
+def oo_list_to_dict(lst, separator='='):
```

Comment:
Let's not do this.  Let's have the user give us a dictionary and create a new variable.  Since this isn't actually used by docker, no need to re-use docker's variables.

Reply:
This is used by docker.  it is used by `daemon.json` is the template for `/etc/docker/container-daemon.json` which is used by docker when running in a container.  So in this case `log-opts` is exactly the same as `openshift_docker_log_options`

Reply:
Okay, that sounds good then.  I think in the future we should do the reverse, we've talked about the current inflexibility of the docker options as impemented.  I think it would be better as a list of kv pairs, and we convert a deprecated list k=v.